Sources/TuistGenerator/Settings/DefaultSettingsProvider.swift

```diff
@@ -158,6 +158,10 @@ public final class DefaultSettingsProvider: DefaultSettingsProviding {
             )
         }
 
+        if target.supportsCatalyst {
+            settings.overlay(with: ["CODE_SIGN_IDENTITY": "-"], for: .macOS)
+        }
```

Comment:
Why is this needed?

Reply:
This approach sets the default `CODE_SIGN_IDENTITY` for the macOS destination to 'Sign to Run Locally'. I noticed similar logic in `XcodeProj`, which allows consumers to run the project directly without specifying `CODE_SIGN_IDENTITY`. Do you think this logic should be moved into `XcodeProj`?

Reply:
No, I think it's fine to keep it here. Thanks for the explanation!

Reply:
Can you add a comment to the source code with the explanation, so it's obvious to anybody else who reads that piece in the future?

Reply:
Sure, I’ll add a comment to clarify the purpose for future readers.